IBAnimatable/GradientDesignable.swift

```diff
@@ -354,3 +356,23 @@ public extension GradientDesignable where Self: UIView {
     }
   }
 }
+
+private class GradientView: UIView {
+
+  // MARK: - Life cycle
+  
+  init(frame: CGRect, layer: CAGradientLayer) {
```

Comment:
Do we need this `init` with layer method here? can we just remove both `init` and add
```
gradientView.autoresizingMask = [.FlexibleWidth, .FlexibleHeight]
gradientView.layer.insertSublayer(gradientLayer, atIndex: 0)
```
between line 62 and 63?